easybuild/README.md

```diff
@@ -1,7 +1,35 @@
 # EasyBuild Examples
 
- TODO: write me
+CCR staff utilize the Easybuild framework when installing software and this tool is made available to CCR users to install software themselves. Easybuild ensures that the compilation and linking that happens during a software installation is done correctly and will work with our systems. CCR staff do not offer courses on using Easybuild, nor are we able to troubleshoot issues you might run into. However, there is excellent [documentation](https://docs.easybuild.io/) and [tutorials](https://tutorial.easybuild.io/) provided by Easybuild developers.
 
-## How to use
+This directory includes examples for building and rebuilding software with EasyBuild on CCR's clusters. For background information on EB recipes, toolchains, and related concepts, refer to our [EasyBuild documentation](https://docs.ccr.buffalo.edu/en/latest/howto/easybuild/#what-is-a-recipe). For common questions and troubleshooting tips, check out our EasyBuild FAQ [here](https://docs.ccr.buffalo.edu/en/latest/howto/easybuild/#frequently-asked-questions).
 
-TODO: write me
+## When to use or not use EasyBuild to install software in your account
+
+There are a few use cases in which you may want to use EasyBuild to install software in your own space (project space preferred over home directories):
+
+1. If you need a custom or modified build
+2. If you need to install a nightly build, or a version of a software which does not have a release number
+3. If there isn't an existing EB recipe and you want to create your own
+4. If we at CCR are not allowed to install the package centrally for licensing reasons, such as some commercial software packages (Gaussian, Orca, VASP and Materials Studio in particular)
+
+On the contrary, you should not install software packages in your own space for the following reasons:
```

Comment:
I would combine this sentence with the second in the list and possibly make this an alert:
We do NOT recommend users attempt to build their own compilers or toolchains.  CUDA builds on our systems require expert intervention.  Therefore, we recommend rather than building anything with CUDA, users utilize pre-built [NVIDIA containers](https://docs.ccr.buffalo.edu/en/latest/howto/containerization/#example-gpu-container-workflow).